matlab/library/WBToolboxLibrary_repository.mdl

```diff
@@ -3184,6 +3206,165 @@ Library {
 		}
 	      }
 	    }
+	    Block {
+	      BlockType		      SubSystem
+	      Name		      "RelativeJacobian"
+	      SID		      "1862"
+	      Ports		      [1, 1]
+	      Position		      [170, 193, 330, 247]
+	      ZOrder		      71
+	      RequestExecContextInheritance off
+	      ContentPreviewEnabled   on
+	      Object {
+		$PropName		"MaskObject"
+		$ObjectID		124
+		$ClassName		"Simulink.Mask"
+		Type			"Relative Jacobian"
+		Description		"This block retrieves the Relative Jacobian between two specified frames.\n\nAssuming DoFs is the numbe"
+		"r of internal degrees of freedom of the robot:\n\nInput:\n  - Joint configuration: Vector of size DoFs, representing"
+		" the configuration of the joints.\n\nOutput:\n  - Jacobian: a 6xdofs matrix representing the Jacobian between the sp"
+		"ecified frames written in Mixed representation: origin as child, orientation as parent.\n\nParameters:\n  - Frame na"
```

Comment:
I may be missing something, but it is not clear from here wheter "Frame 1" is parent or "Frame 2" is child. Furthermore, I am not sure if parent/child are the best choice of words, as this frames can be also relative to links that are not one the parent of another. Perhaps referenceFrame/frame or reference/leaf could be better names?

Reply:
Yes!

![image](https://user-images.githubusercontent.com/45564317/151189492-dee94066-1d35-4d09-a6f6-b27702abddea.png)

Reply:
Updated in https://github.com/robotology/wb-toolbox/pull/226/commits/c70ea7d4a826fc7d2fdd37aaf7a1634cf7033ec2